apps/webapp/vite.config.ts

```diff
@@ -17,4 +17,7 @@ export default defineConfig({
       },
     },
   },
+  test: {
+    environment: 'jsdom',
```

Comment:
Needed this because `webapp` references `TransactionToast` in its tests. `TransactiionToast`, in turn, uses React elements (for the progress bar and the `<Link />` to the transaction details). As a result, webapp tests were breaking since they didn't know how to handle DOM elements.